AlphaWallet/Rpc Network/AvailableRpcNetwork.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  AvailableRpcNetwork.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 20/12/21.
+//
+
+import Foundation
+
+@available(iOS 13.0, *)
+fileprivate let SelectedCompressionAlgorithm = NSData.CompressionAlgorithm.lzma
+
+fileprivate let ChainsZipFileName = "chains_mini.zip"
+fileprivate var chains: [CustomRPC]?
+
+struct RpcNetwork: Codable {
+    let name: String
+    let chainId: Int
+    let shortName: String
+    let networkId: Int
+    let nativeCurrency: NativeCurrency
+    let rpc: [String]
+    let faucets: [String]
+    let infoURL: String
+
+    var isTestNet: Bool {
```

Comment:
The source `.json` files has a `network` field. Can we use that to tell if it's mainnet/testnet (careful that the value might not be standardize, eg. "mainnet" vs "Mainnet") and then fallback to scanning `name` if we have to?

```
$ ag \"network\" | head
eip155-888.json:4:  "network": "mainnet",
eip155-71393.json:4:  "network": "testnet",
eip155-172.json:4:  "network": "testnet",
eip155-66.json:4:  "network": "mainnet",
eip155-42220.json:6:  "network": "Mainnet",
eip155-31.json:4:  "network": "testnet",
eip155-200625.json:4:  "network": "mainnet",
eip155-365.json:4:  "network": "testnet",
eip155-80001.json:4:    "network": "testnet",
eip155-1313500.json:4:  "network": "mainnet",
```

Reply:
The mini json file choosen did not have this field. Do you want me to use the other file?

Reply:
Sorry missed this one. Yes